documentation/book/ref-storage-jbod.adoc

```diff
@@ -34,10 +40,13 @@ storage:
 # ...
 ----
 
-It is not possible to change the `id` property for a volume.
-It is not possible to add or remove volumes from a JBOD storage configuration.
+Do not change the `id` property of a JBOD volume.
```

Comment:
Tbh the statement I used sounded clearer than this one. We should say that after volumes creation with some ids, they cannot be changed anymore.

Reply:
Thanks - I will update.